packages/ui-components/src/assets/svg-symbols.svg

```diff
@@ -1420,4 +1420,29 @@
 		<path d="M6.98871 3.34583c-6.651616 3.83473-6.651613 13.47357 0 17.30837C13.6403 24.4889 22 19.6695 22 12 22 4.33053 13.6403-.488906 6.98871 3.34583Zm1.9009 14.61327c-4.77108 0-.77848-5.3646-4.09281-5.2721v-1.4336c3.12651.3879-.6012-5.71145 4.06968-5.17962v1.10991c-2.60521.06167-.01541 3.39141-2.35856 4.78651 2.35856 1.5492-.12333 5.0794 2.38169 4.694v1.2949Zm-.16741-5.2118c-.53833-.3103-.53833-1.0904 0-1.4008.53832-.3103 1.21488.0797 1.21488.7004s-.67656 1.0107-1.21488.7004Zm2.8722 0c-.5383-.3103-.5383-1.0904 0-1.4008.5384-.3103 1.2149.0797 1.2149.7004s-.6765 1.0107-1.2149.7004Zm2.8723 0c-.5384-.3103-.5384-1.0904 0-1.4008.5383-.3103 1.2148.0797 1.2148.7004s-.6765 1.0107-1.2148.7004Zm.6437 5.2118v-1.2949c2.505.3854.0231-3.1448 2.3817-4.694-2.3432-1.3951.2466-4.72484-2.3586-4.78651V6.07378c4.6709-.53183.9432 5.56752 4.0697 5.17962v1.4336c-3.3143-.0925.6783 5.2721-4.0928 5.2721Z"/>
 	</symbol>
 
+	<symbol id="kv-monitoring"
+		xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+		<mask id="a" width="24" height="24" x="0" y="0" maskUnits="userSpaceOnUse" style="mask-type:alpha"><path fill="#D9D9D9" d="M0 0h24v24H0z"/></mask><g mask="url(#a)"><path d="M3 21v-2l2-2v4H3Zm4 0v-6l2-2v8H7Zm4 0v-8l2 2.025V21h-2Zm4 0v-5.975l2-2V21h-2Zm4 0V11l2-2v12h-2ZM3 15.825V13l7-7 4 4 7-7v2.825l-7 7-4-4-7 7Z"/></g>
```

Comment:
Is this mask necessary?

Reply:
Built without this masks and it seems it works just fine